Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -181,9 +235,10 @@ ParserQuery: |
       DnsResponseCode=toint(DnsResponseCode),
       SrcPortNumber=toint(Port_s),
       DvcHostname=Dvc,
+      DvcIpAddr=iff(HostIP_s == "","0.0.0.0",HostIP_s),
```

Comment:
The schema says that `DvcIpAddr` is a recommended field to have, and the old version of the parser did not fill in that value, so I condluded that it would create value to provide a meaningful value for that field.
As to why I defaulted to using 0.0.0.0 when the information is not available, I thought that having a valid IP address in there would be better than having a null value or an empty value.
But if it makes more sense we can use either null or empty string. But in that case having the if structure is pointless (if we want to use empty string for the empty state, then there is no need to check HostIP_s, we can just copy its value in either case):
```
DvcIpAddr=HostIP_s
```

Reply:
DvcIpAddr=HostIP_s would be fine

Reply:
Okay, I will update it.